test/dash/dash_parser_manifest_unit.js

```diff
@@ -1844,6 +1845,11 @@ describe('DashParser Manifest', () => {
       '      </Representation>',
       '    </AdaptationSet>',
       '    <AdaptationSet contentType="image" id="3">',
+      '      <SegmentTemplate media="$Number$.jpg" ',
+      '        duration="2" startNumber="1"/>',
+      '      <Representation id="thumbnails" width="1024" height="1152">',
+      '        <EssentialProperty schemeIdUri="' + idUri + '" value="10x20"/>',
```

Comment:
In current versions of Shaka Player, you are allowed to use template strings in backticks like:

```js
`    <EssentialProperty schemeIdUri="${thumbnailScheme}" value="10x20" />`,
```

Reply:
It seems that in testing this is not allowed, I see the following error:
`  1851:7  error  Unexpected template string expression                no-template-curly-in-string`

Reply:
If you update the eslint policy in the master branch, I can include this in this PR, but for the moment I will have to leave it like that.

Reply:
The only thing is that the line is more than 80 characters long, so I'll add an eslint exception to make everything easier to read.